portfolio/src/main/webapp/assets/js/main.js

```diff
@@ -0,0 +1,219 @@
+/*
+	Big Picture by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+*/
+
+(function($) {
+
+	var	$window = $(window),
+		$body = $('body'),
+		$header = $('#header'),
+		$all = $body.add($header);
+
+	// Breakpoints.
+		breakpoints({
+			xxlarge: [ '1681px',  '1920px' ],
+			xlarge:  [ '1281px',  '1680px' ],
+			large:   [ '1001px',  '1280px' ],
+			medium:  [ '737px',   '1000px' ],
+			small:   [ '481px',   '736px'  ],
+			xsmall:  [ null,      '480px'  ]
+		});
+
+	// Play initial animations on page load.
+		$window.on('load', function() {
+			setTimeout(function() {
+				$body.removeClass('is-preload');
+			}, 100);
+		});
+
+	// Touch mode.
+		if (browser.mobile)
+			$body.addClass('is-touch');
+		else {
+

```

Comment:
if this was auto-added don't worry about it, but all this line spacing is too much